src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/ViewModels/GeneralViewModel.cs

```diff
@@ -506,7 +506,7 @@ private void UpdateNowClick()
             IsNewVersionDownloading = _isDownloadButtonClicked && string.IsNullOrEmpty(UpdatingSettingsConfig.DownloadedInstallerFilename);
             NotifyPropertyChanged(nameof(IsDownloadAllowed));
 
-            Process.Start(new ProcessStartInfo("powertoys://update_now") { UseShellExecute = true });
+            Process.Start(new ProcessStartInfo(Helper.GetPowerToysInstallationFolder() + "\\PowerToys.exe") { Arguments = "powertoys://update_now/" });
```

Comment:
- Looks like sn incorrect argument. Shouldn't it be only `update_now`?

Reply:
@yuyoyuppe 
should it be `powertoys://update_now/` of `-update_now`?

Reply:
@enricogior
Is the first `\` in `\\PowerToys.exe` an escape character or do we need `\PowerToys.exe` here?

Reply:
It's an escape char

Reply:
we're using `powertoys://update_now/` to have a unified flow between toast notifications buttons, which still use URIs, and button on the settings page.

Reply:
can we add a describing comment about this fact